api/api.go

```diff
@@ -207,6 +207,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 		router.GET("/renter", api.renterHandlerGET)
 		router.POST("/renter", RequirePassword(api.renterHandlerPOST, requiredPassword))
 		router.GET("/renter/contracts", api.renterContractsHandler)
+		router.GET("/renter/contract/*contractid", api.renterContractDetailHandler)
```

Comment:
>Is there a reason not to expose everything in `modules.RenterContract`?

The main reason is that it's easy to add new fields to an API, but hard to remove them. So we prefer to be as conservative as possible when adding new fields. That's why we typically define new types local to the `api` package that are subsets of the types passed around internally.